src/assessments/seo/textCompetingLinksAssessment.js

```diff
@@ -1,73 +1,116 @@
-var AssessmentResult = require( "../../values/AssessmentResult.js" );
+const Assessment = require( "../../assessment.js" );
+const AssessmentResult = require( "../../values/AssessmentResult.js" );
 
-var Mark = require( "../../values/Mark.js" );
-var addMark = require( "../../markers/addMark.js" );
+const Mark = require( "../../values/Mark.js" );
+const addMark = require( "../../markers/addMark.js" );
 
-var map = require( "lodash/map" );
+const map = require( "lodash/map" );
+const merge = require( "lodash/merge" );
+const isUndefined = require( "lodash/isUndefined" );
 
 /**
- * Returns a score and text based on the number of links.
- *
- * @param {object} linkStatistics The object with all linkstatistics.
- * @param {object} i18n The object used for translations
- * @returns {object} resultObject with score and text
+ * Assessment to check whether the keyphrase is encountered in the first paragraph of the article.
```

Comment:
Wrong class. `Assessment to check whether you're linking to a different page with the focus keyword from this page.` or something like that.

Reply:
🤦‍♀️ Fixed.